src/concrete/ml/sklearn/base.py

```diff
@@ -1328,6 +1335,80 @@ def __init__(self, n_bits: Union[int, Dict[str, int]]):
 
         BaseEstimator.__init__(self)
 
+    @classmethod
+    def from_sklearn_model(
+        cls,
+        sklearn_model: sklearn.base.BaseEstimator,
+        X: Optional[numpy.ndarray] = None,
+        n_bits: int = 8,
```

Comment:
so 8 bits is ok? I remember it wasn't always the best choice in the graphs and we had discussed 9 bits. What is the accuracy on the regressors ?